src/Nancy/Responses/DefaultXmlSerializer.cs

```diff
@@ -64,19 +77,19 @@ public void Serialize<TModel>(MediaRange mediaRange, TModel model, Stream output
             }
         }
 
-        private static bool IsXmlType(string contentType)
+        private static bool IsXmlType(MediaRange mediaRange)

```

Comment:
What's the reason for this? This means we're ToString'ing the `mediaRange` twice here. Once in the `IsNullOrEmpty` implicit conversion and once below.